Hearthstone Deck Tracker/HsLogReader.cs

```diff
@@ -424,10 +424,15 @@ private void Analyze(string log)
 							Game.Entities.FirstOrDefault(e => e.Value.HasTag(GAME_TAG.PLAYER_ID) && e.Value.GetTag(GAME_TAG.PLAYER_ID) == Game.OpponentId);
 
 						var match = _actionStartRegex.Match(logLine);
-						var id = match.Groups["Id"].Value.Trim();
```

Comment:
Does this compile for you? The renaming of `id` to `actionStartingCardId` causes build errors for me on lines 482, 487 & 492 those lines still use `id`.